src/email-templates/account/magiclink-email.pug

```diff
@@ -1,10 +1,10 @@
 doctype html
-html
-  body(style='margin: 0;font-family: Tahoma;font-size: 1.2em;background-color: #ECF0F1;')
-    header.text-center(style='padding: 1em;background-color: #34495E;margin-bottom: 1em;text-align: center;')
-      .logo(style='max-width: 33.33%;')
-        img(src=logo)
-    .body(style='padding: 1em;margin-right: auto;margin-left: auto;max-width: 500px;color: #2C3E50;')
-      h1(style='font-family: Palatino;font-size: 1.5em;') Hello
-      p Please click the below link to sign in.
-      a.reset-btn(href=hashLink, style='color: #2980B9;') #{hashLink}
+body(style='margin: 0;')
+  header
+    img(src=logo, style='display: block; margin: 0.5em; max-width: 6em;')
```

Comment:
Minor problem that the current logo is white, and the email's background is white, so, won't show up. We'll fix up the logo later, or change background.